homeassistant/components/pooldose/sensor.py

```diff
@@ -0,0 +1,212 @@
+"""Sensors for the Seko Pooldose integration."""
+
+from __future__ import annotations
+
+from collections.abc import Callable
+from dataclasses import dataclass
+import logging
+from typing import Any
+
+from homeassistant.components.sensor import (
+    SensorDeviceClass,
+    SensorEntity,
+    SensorEntityDescription,
+)
+from homeassistant.const import EntityCategory
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_platform import AddConfigEntryEntitiesCallback
+
+from . import PooldoseConfigEntry, PooldoseCoordinator
+from .const import device_info
+from .entity import PooldoseEntity
+
+_LOGGER = logging.getLogger(__name__)
+
+
+@dataclass(frozen=True, kw_only=True)
+class PooldoseSensorEntityDescription(SensorEntityDescription):
+    """Describe a Pooldose sensor entity."""
+
+    entity_category: EntityCategory | None = None
+    value_fn: Callable[[Any], Any] = lambda data: data[0] if data else None
+
+
+SENSOR_DESCRIPTIONS: tuple[PooldoseSensorEntityDescription, ...] = (
+    PooldoseSensorEntityDescription(
+        key="temperature",
+        translation_key="temperature",
+        device_class=SensorDeviceClass.TEMPERATURE,
+    ),
+    PooldoseSensorEntityDescription(
+        key="ph",
+        translation_key="ph",
+        device_class=SensorDeviceClass.PH,
```

Comment:
so there's a thing called device class translations, where if the name of the device class matches the expected name of the entity, you can just omit the translation key and icon, and it will show the default. This saves our translators a few thousand "temperature"s to translate :D